### PR TITLE
Dns adapter

### PR DESCRIPTION
**Pull request by pelger**
DNS service discovery adapter.

### CODE REVIEW

**Comment by mcollina** [comment]
Is this based on #74 ?

**Comment by pelger** [comment]
Yep - local merge with 74 - waiting for you guys to complete review on 74 before merging into master

File: test/adapters/support/dns.stub.js [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+ * HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+ * STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING
+ * IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+
+'use strict'
+
+var FugeDns = require('fuge-dns')
```
Comment (mcollina) [inline test/adapters/support/dns.stub.js]: this should be published before merging.

File: test/drivers/http-driver/500error.test.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+ * IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+
+'use strict'
+
+var http = require('http')
+var test = require('tap').test
+var muHttp = require('../../../packages/mu-http')
+var Mu = require('../../../packages/mu')
```
Comment (mcollina) [inline test/drivers/http-driver/500error.test.js]: check the resolving

**Pull request merged**